app/workers/github/create_issue_job.rb

```diff
@@ -4,21 +4,15 @@
 module Github
   class CreateIssueJob
     include Sidekiq::Worker
-
-    # limit the rate at which Github API calls are made
-    # so as not to trigger Github spam protections
-    sidekiq_options queue: 'low',
-                    rate: {
-                      name: 'prevent_feedback_spam',
-                      limit: 20,
-                      period: 3600, ## An hour
-                    }
+    THROTTLE = Sidekiq::Limiter.window('prevent_feedback_spam', 4, :minute).freeze
 
     # :nocov:
     def perform(feedback)
-      feedback = Feedback.new(feedback)
-      create_response = Github::GithubService.create_issue(feedback)
-      FeedbackSubmissionMailer.build(feedback, create_response&.html_url).deliver_now
+      THROTTLE.within_limit do
+        feedback = Feedback.new(feedback)
+        create_response = Github::GithubService.create_issue(feedback)
+        FeedbackSubmissionMailer.build(feedback, create_response&.html_url).deliver_now
```

Comment:
Should we use `deliver_later` so we don't end up creating two GH issues in case this fails and the job is retried?

Reply:
Seems like a good idea to me.  Will that _just work_ without any config?

Reply:
No, needs some additional config... It's not absolutely necessary to have as part of this PR, but set up #7111 and #7112 to track this change.